### PR TITLE
SRCH-4117 Update search-gov README about using docker

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- Updated ReadMe for using search-gov within docker.
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

**Comment by krbhavith** [comment]
@lorawoodford can you please review the PR.

**Comment by lorawoodford** [comment]
@krbhavith This seems to be doing way more than just updating the readme.  Is that intentional?  I'm heading out for the day, but can given this more time tomorrow, but just wanted to note that this includes way more than the PR title and description suggest it should.

**Comment by krbhavith** [comment]
@lorawoodford it had code changes from [docker development](https://github.com/GSA/search-gov/pull/1186), now rebased with master.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,17 @@ Use [NVM](https://github.com/nvm-sh/nvm#installing-and-updating) to install the
 
 The required services (MySQL, Elasticsearch, etc.) can be run using Docker. Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for services.
 
+`search-gov` application can be in local machine or using Docker container which is configured to run on the port [3100](http://localhost:3100/). Few dependencies ([Ruby](https://github.com/GSA/search-gov#ruby), [NodeJS](https://github.com/GSA/search-gov#nodejs), [Package Manager](https://github.com/GSA/search-gov#package-manager), [Packages](https://github.com/GSA/search-gov#packages), [Gems](https://github.com/GSA/search-gov#gems), [JavaScript dependencies](https://github.com/GSA/search-gov#javascript-dependencies)) required to run application are installed using Docker. However, other dependent data and configuration needs to be setup manually, which can be done with in the running container using `bash` or in local machine.
+
+Using bash to perform any operations on search-gov application running in Docker container.
+
+    $ docker compose run search-gov bash
```
Comment (ddzz) [inline README.md:29]: We should specify that this command needs to be run in `search-services`.

**Pull request merged**